src/index.css

```diff
@@ -1,74 +1,172 @@
+/* this file contains global CSS styles that apply to the entire application */
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+.landing-page .overlap {
```

Comment:
Optional - If the landing page is Home.jsx, this should ideally go along with the Home.css